src/components/SharedComponents/TaxonResult.tsx

```diff
@@ -141,20 +183,16 @@ const TaxonResult = ( {
       }
       testID={testID}
     >
-      <Pressable
+      <TaxonResultMain
         className={
           classnames( "flex-row items-center shrink", {
             "py-3": asListItem
           } )
         }
-        onPress={handleTaxonOrEditPress || navToTaxonDetails}
-        accessible
-        accessibilityRole="link"
-        accessibilityLabel={accessibleName}
-        accessibilityHint={t( "Navigates-to-taxon-details" )}
       >
         <View className="w-[62px] h-[62px] justify-center relative">
           <ObsImagePreview
+            // TODO fix when ObsImagePreview typed
```

Comment:
What needs to be fixed?
Will it be enough to grep the codebase for the class name whenever something is typed, or do we need another mechanism for finding a TODO of this kind?

Reply:
ObsImagePreview is in Javascript but has flow types, and those flow types are not great. They claim that the `source` prop must have a `uri` attribute, even though that component explicitly works around a missing source URI and adds a placeholder image. If you follow and fix all these problems when doing piecemeal TypeScript conversion, you end up converting the whole repo, which would be way out of scope.

As to what to do about fixing these problems if/when the underlying problem is fixed, IDK. I just assume if someone sees a message like this and there's no associated linter warning, they delete the comment, or they fix the problem.